### ISSUE TITLE
Verify page: "Error: no header found"

### ISSUE THREAD

**Issue by dabura667** [keybase/client#8029]
https://keybase.io/verify

When inserting a saltpack signed message, it always gives this error.

BEGIN KEYBASE SALTPACK SIGNED MESSAGE. kXR7VktZdyH7rvq v5weRa0zkDbcHiZ z1KmaVGP2MwDu8W IeOGWy46RgXq0vo vyO9lixKuQEJJ5b fHHInZJvGEdYcR1 LSOPcF4y6CV5Fta acr9eYCg7nsn5Zh wvmHAXhD07xJ25i fyH9kLZUBGheVJT 0xfAUDxiCpRViPm dGwZX8zQbjPfEFd mtzEA6WMqTdOGs1 . END KEYBASE SALTPACK SIGNED MESSAGE.

**Comment by maxtaco** [comment]
Yeah, that's confusing, those pages predate saltpack and are just for PGP.  We should fix it

cc: @malgorithms

**Comment by dabura667** [comment]
Ideally, you should detect and verify saltpack, and the sign page should have a switch for pgp or saltpack.

The saltpack format is pretty cool imo, and would like to see it easier to sign and verify etc. via the browser.

**Comment by GeorgeBgk** [comment]
BTW Problem still persists. @malgorithms

**Comment by seanhelling** [comment]
Encountered this today. My use case is allowing visitors to my personal website to verify my SSH public key.

**Comment by dhda** [comment]
I'd also like to see the verification page support both PGP and saltpack. I want to be able to verify to my Facebook friends (same use as for any platform that doesn't explicitly work with the proof system) that my Keybase account is actually me by posting a signed message that they can then verify. It's important for people to be able to verify my signature and my Keybase identity without having to use and install Keybase themselves, and https://keybase.io/verify is a hugely useful tool for this kind of real world interoperability. Bonus is that it would be a tool that makes adopting Keybase much much smoother!

From the Keybase apps (desktop Crypto tools, and the new mobile Crypto tools) I can generate saltpack signatures, which the web verify page doesn't support. From the apps I also *can't* do anything with PGP, which would be another useful feature so that all these tools are consistent with one another.

**Comment by Kuchenm0nster** [comment]
Bump
Still PGP

**Comment by alexparker** [comment]
Bumpity bump bump

**Comment by ryanbas** [comment]
I ran into this issue today. The error should at least say the header is unsupported and not that the header doesn't exist.

**Comment by megavillain** [comment]
Thought I was going crazy.. turns out it's still using PGP, thankfully google lead me here.

**Comment by devbanana** [comment]
Another vote for this. Just started using keybase and this would definitely be helpful.
